notifications_utils/clients/zendesk/zendesk_client.py

```diff
@@ -76,27 +76,111 @@ def create_ticket(
                 'name': user_name or '(no name supplied)'
             }
 
+        self.send_ticket_data_to_zendesk(data)
+
+    def send_ticket_data_to_zendesk(self, data):
```

Comment:
Optional

I've done a little more thinking about this, in particular now I've started reviewing the admin PR that brings this in.

At the moment in your PR we have

ZendeskClient with a method `send_ticket_data_to_zendesk` which is given a dictionary of `data`.

This means when you use it in the admin app you say `zendesk_client.send_ticket_data_to_zendesk(ticket.request_data)`.

The bit I'm not as keen on is everytime we want to send a ticket we need to remember to give it `ticket.request_data` rather than just the ticket.

One way we could do this would be to give ZendeskClient a method `send_ticket_to_zendesk` which takes a `NotifySupportTicket` and then calls `self.send_ticket_data_to_zendesk(ticket.request_data)`. 

Note, this is all optional. I think it would be an improvement but I'm not 100% sure on that and even if you agree it is, I'm open to us moving quicker and merging what we've got so far anyway

Reply:
@idavidmcdonald I've made this change - added in a new commit d3165e3